bin/tictactoe

```diff
@@ -1,3 +1,13 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to TicTacToe!"
+puts "Would you to like to play in Standard Mode or Wargames Mode?"
+puts "Select standard or wargames."
+input = gets.chomp
+if input == "wargames"
+  Game.war 
+else
+  Game.start 
+end
```

Comment:
Great bin file, short length is ideal. Also, love the flair with the wargames option!  Nice touch!  We would, however, prefer to see the CLI logic separated from the game logic in its own class (e.g. TicTacToeCLI). Splitting the interface from the implementation of the game allows for multiple interfaces (e.g. a web app) without requiring changes to the game code.